### DISCUSSION TITLE
机器人能接受无法发送

### DISCUSSION THREAD

**Discussion by ecjcjdjfn** [nonebot/discussions#465] [问答] [answered]
![11141](https://github.com/nonebot/discussions/assets/57409755/980a018f-38c5-43f1-b81a-ebad6c78563b)

加载了历史上的今天这个插件，别的插件也试了。连上反向地址了，能收到消息但是机器人不会发送消息。
![5555555224934](https://github.com/nonebot/discussions/assets/57409755/6b614147-010a-4d73-bb79-d1f596d8cfd3)

**Comment by yanyongyu** [comment, marked as answer]
从你的截图里并没有看见加载了历史上的今天插件，并且也没看见你使用该插件指令的消息日志

**Reply by ecjcjdjfn** [reply]
![111202441](https://github.com/nonebot/discussions/assets/57409755/0d531617-639e-4a1d-89f9-fb127c96d161)

**Reply by yanyongyu** [reply]
你还是没有加载这个插件啊？

**Reply by ecjcjdjfn** [reply]
![QQ图片20240515204446](https://github.com/nonebot/discussions/assets/57409755/a2970deb-b0ab-4eaa-add8-8f1ad63835c2)

**Reply by yanyongyu** [reply]
请把报错截全

**Reply by ecjcjdjfn** [reply]
![QQ图片20240515204856](https://github.com/nonebot/discussions/assets/57409755/c9885ec7-e2e4-441e-ac48-b132ce0fbb54)

**Reply by yanyongyu** [reply]
看报错是插件使用了ujson这个库但是你的环境似乎有点问题，你可以尝试重新安装ujson

**Reply by ecjcjdjfn** [reply]
好的谢谢 我在试试看看

**Reply by ecjcjdjfn** [reply]
![QQ图片20240515230124](https://github.com/nonebot/discussions/assets/57409755/5d8638dc-497a-4c35-98ae-0baf708724c3)
弄了半天是VC+组件没有， 现在应该都是加载了吧，还是不会回复消息。
![0606EA92](https://github.com/nonebot/discussions/assets/57409755/df24bacc-07de-4813-b459-4ff998e7908d)

**Reply by yanyongyu** [reply]
如果你没有修改 command start 配置项你应该用 `/` 开头使用命令

**Reply by ecjcjdjfn** [reply]
![QQ图片20240515231028](https://github.com/nonebot/discussions/assets/57409755/ea5682ba-07f9-407e-a99a-b0c1df8ffa5f)
哇可以了 谢谢大佬。没看到哪里说了要/开头啊
